test.cpp

```diff
@@ -1257,6 +1286,65 @@ void simplifyPath()
     ASSERT_EQUALS("/../1.c", simplecpp::simplifyPath("/../a/../1.c"));
 }
 
+// tests transferred from cppcheck
+// https://github.com/danmar/cppcheck/blob/d3e79b71b5ec6e641ca3e516cfced623b27988af/test/testpath.cpp#L43 ff.
+void simplifyPath_cppcheck()
```

Comment:
I did not see the "sorting" of the testcases making it hard to see what corner cases are covered. or if there are duplicate tests.

do we need all tests with // ? that is 1 condition in the code that is tested with 17 assertions. if those are needed I would suggest we group all equivalent tests together. a/b a//b a///b are put together. then when reading those one does not need to re-interpret the path every time.